### PR TITLE
Notifications update

### PR DESCRIPTION
**Pull request by ashonuyak**
image

### CODE REVIEW

**Comment by oleksandr-danylchenko** [comment]
@R0ixy @iryb Help me please :pray:

**Comment by iryb** [comment]
@ashonuyak You should base your PR on https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/307 or wait until it will be merged. I can't test it locally.

**Comment by iryb** [comment]
@ashonuyak Notifications work fine. Please resolve the conflicts.

**Comment by ashonuyak** [comment]
@iryb done

**Review by iryb — DISMISSED** [review]

File: frontend/src/contexts/compare-context.tsx
Comment (R0ixy) [inline frontend/src/contexts/compare-context.tsx]: We need to show notification only if user _really_ added car to comparison. Beacuse user can be unauthorized and click on this button. In this case he will be redirected to sign-in but notification still appears.

I think we need to move this logic to `handleAddToCompare`. To RTKQ result promise on line 48

**Review by R0ixy — CHANGES_REQUESTED** [review]
Please fix this https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/300#discussion_r969836321

and I think it can be merged

**Comment by oleksandr-danylchenko** [comment]
Thanks for pointing this out, @R0ixy :raised_hands:

**Review by R0ixy — APPROVED** [review]
LGTM

**Review by oleksandr-danylchenko — APPROVED** [review]

**Pull request merged**
